examples/engine_headless.rs

```diff
@@ -96,7 +96,7 @@ fn main() -> Result<(), Box<dyn std::error::Error>> {
         // We create a headless Universal engine.
         let engine = EngineBuilder::headless();
         let mut store = Store::new(engine);
-        let mut env = FunctionEnv::new(&mut store, ());
+        let _env = FunctionEnv::new(&mut store, ());
```

Comment:
If it's not used, we can just remove this line :)

Reply:
done